### PR TITLE
Improve text-generation example ipex installation

### PR DESCRIPTION
**Pull request by changwangss**
## Type of Change

1. set config.use_cache to True.
2. improve ipex installation.
    - 0711+cpu whl has been removed by official torch. https://download.pytorch.org/whl/nightly/cpu
    - add env to support mpt/neural-chat int8 workaround.
## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: examples/huggingface/pytorch/text-generation/quantization/README.md [hunk 1]
```diff
@@ -17,15 +17,15 @@ WORK_DIR=$PWD
...
 conda create -n llm python=3.9 -y
-# install deps
+# install deps, please try gcc, gxx 12.2 if 12.3 doesn't find from conda
 conda install gcc=12.3 gxx=12.3 cxx-compiler -c conda-forge -y
 conda install cmake ninja mkl mkl-include -y
 conda install gperftools -c conda-forge -y
 
 # Install PyTorch
-python -m pip install torch==2.1.0.dev20230711+cpu torchvision==0.16.0.dev20230711+cpu torchaudio==2.1.0.dev20230711+cpu --index-url https://download.pytorch.org/whl/nightly/cpu
+python -m pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cpu
```
Comment (hshen14) [inline examples/huggingface/pytorch/text-generation/quantization/README.md:26]: Is nightly stable? it's also dynamic. Better to use a fixed stable version.

File: examples/huggingface/pytorch/text-generation/quantization/README.md [hunk 2]
```diff
@@ -37,13 +37,17 @@ make install -j$(nproc)
...
-git clone --branch llm_feature_branch https://github.com/intel/intel-extension-for-pytorch.git
+git clone https://github.com/intel/intel-extension-for-pytorch.git
 cd intel-extension-for-pytorch
 git submodule sync && git submodule update --init --recursive
 export DNNL_GRAPH_BUILD_COMPILER_BACKEND=1
 export CXXFLAGS="${CXXFLAGS} -D__STDC_FORMAT_MACROS"
 python setup.py install
 cd ../
+
+# it is necessary for mpt and neural-chat-v1-1 models, other models don't need it.
```
Comment (kevinintel) [inline examples/huggingface/pytorch/text-generation/quantization/README.md:48]: it's better to give link of this env

File: examples/huggingface/pytorch/text-generation/quantization/README.md (hunk 1 above)
Comment (changwangss) [reply examples/huggingface/pytorch/text-generation/quantization/README.md]: change another 0711 link to replace.

File: examples/huggingface/pytorch/text-generation/quantization/README.md (hunk 2 above)
Comment (changwangss) [reply examples/huggingface/pytorch/text-generation/quantization/README.md]: updated the comment.

**Review by hshen14 — APPROVED** [review]

**Review by kevinintel — APPROVED** [review]

**Pull request merged**